saas/axops/src/applatix.io/axops/configuration.go

```diff
@@ -83,52 +83,70 @@ func GetConfigurationsByUser() gin.HandlerFunc {
 // @Failure 500 {object} axerror.AXError "Internal server error"
 // @Resource /configurations
 // @Router /configurations/{user}/{name} [GET]
-func GetConfigurationsByUserName() gin.HandlerFunc {
+func GetConfiguration() gin.HandlerFunc {
 	return func(c *gin.Context) {
 		username := c.Param("user")
 		name := c.Param("name")
-		configList, axErr := configuration.GetConfigurationsByUserName(username, name)
+		showSecrets := strings.ToLower(c.Request.URL.Query().Get("show_secrets")) == "true"
+		if showSecrets {
+			user := GetContextUser(c)
+			if !user.IsSuperAdmin() && !user.IsAdmin() {
```

Comment:
Can we replace this check so that it forbids anonymous users only? That way the check is consistent with access policy within the cluster.

Reply:
Ok, I simply have to remove this `if showSecrets` block. anonymous viewing of secrets is already handled by the session token.